ts/Series/Sankey/SankeyColumnComposition.ts

```diff
@@ -150,7 +150,16 @@ namespace SankeyColumnComposition {
                 height += nodeHeight;
                 return height;
             }, 0);
-            return ((series.chart.plotSizeY || 0) - height) / 2;
+
+            // Node alignment option handling #19096
+            switch (series.options.nodeAlignment) {
+                case 'top':
+                    return 0;
+                case 'bottom':
+                    return (series.chart.plotSizeY || 0) - height;
+                default:
+                    return ((series.chart.plotSizeY || 0) - height) / 2;
+            }
```

Comment:
A switch statement like this can be made more compact by an object literal, something like this:

```js
        return ({
            top: 0,
            bottom: 1
        }[series.options.nodeAlignment] ?? 0.5) * ((series.chart.plotSizeY || 0) - height);
```